source/models/the-fixture-adapter.md

```diff
@@ -40,7 +40,7 @@ In order to attach fixtures to your model, you have to use `reopenClass` method
 fixtures:
 
 ```app/models/documenter.js
-let Documenter = DS.Model.extend({
```

Comment:
Only instance of ES6 syntax in the guides. I think it could become a little confusing to those that do not know about ES6.